docs/rules/space-between-siblings.md

```diff
@@ -0,0 +1,123 @@
+---
+pageClass: rule-details
+sidebarDepth: 0
+title: vue/space-between-siblings
+description: Insert newlines between sibling tags in template 
+---
+# vue/space-between-siblings
+
+> Insert newlines between sibling tags in template 
+
+- :exclamation: <badge text="This rule has not been released yet." vertical="middle" type="error"> ***This rule has not been released yet.*** </badge>
+- :wrench: The `--fix` option on the [command line](https://eslint.org/docs/user-guide/command-line-interface#fixing-problems) can automatically fix some of the problems reported by this rule.
+
+## :book: Rule Details
+
+This rule requires newlines between sibling HTML tags
+
+<eslint-code-block fix :rules="{'vue/space-between-siblings': ['error']}">
+
+```vue
+<!-- ✓ GOOD -->
+<template>
+  <div>
+    <ul>
+      <li>
+      </li>
+
+      <li>
+      </li>
+
+      <li>
+      </li>
+    </ul>
+  </div>
+</template>
+```
+
+```vue
+<!-- ✗ BAD -->
+<template>
+  <div>
+    <ul>
+      <li>
+      </li>
+      <li>
+      </li>
+      <li>
+      </li>
+    </ul>
+  </div>
+</template>
+```
+
+</eslint-code-block>
+
+## :wrench: Options
+
+```json
+{
+  "vue/space-between-siblings": ["error", {
+      "ignoreNewlinesBefore": [],
+      "ignoreNewlinesAfter": []
+  }]
+}
```

Comment:
I think the rule should have a primary option `"always" | "never"`, just like https://eslint.vuejs.org/rules/padding-line-between-blocks.html.